test/integration/compoundWrapModule.spec.ts

```diff
@@ -186,13 +194,14 @@ describe("compoundWrapModule", () => {
 
         const underlyingBalance = await setup.dai.balanceOf(setToken.address);
         const wrappedBalance = await cDai.balanceOf(setToken.address);
-
         const delta = preciseMul(setTokensIssued, wrappedQuantity.sub(subjectWrappedTokenUnits));
```

Comment:
Seems like `delta` is calculating the remaining wrapped tokens, which works in this case cause the test is just unwrapping half of the balance but wouldn't be right otherwise. Think `delta` should just be defined as such:

`const delta = preciseMul(setTokensIssued, wrappedQuantity.sub(subjectWrappedTokenUnits));`